lib/twitter-ads/audiences/tailored_audience.rb

```diff
@@ -33,9 +33,9 @@ class TailoredAudience
                            'accounts/%{account_id}/tailored_audience_changes'.freeze # @api private
     RESOURCE_MEMBERSHIPS = "/#{TwitterAds::API_VERSION}/" +
                            'tailored_audience_memberships'.freeze # @api private
-    RESOURCE_USERS       = "/#{TwitterAds::API_VERSION}/ \
-                           accounts/%{account_id}/tailored_audiences/ \
-                           %{id}/users".freeze # @api private
+    RESOURCE_USERS       = "/#{TwitterAds::API_VERSION}/" +
+                           'accounts/%{account_id}/tailored_audiences/' +
```

Comment:
Double quotes for consistency?

Reply:
You need single quotes for string interpolation, double if none. Rubocop's rules

Reply:
Thanks for clarifying. Did not know that